src/components/VideoPopoverMenu/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useRef} from 'react';
+import type {View} from 'react-native';
+import type {PopoverMenuItem} from '@components/PopoverMenu';
+import PopoverMenu from '@components/PopoverMenu';
+import {useVideoPopoverMenuContext} from '@components/VideoPlayerContexts/VideoPopoverMenuContext';
+import type {AnchorPosition} from '@styles/index';
+
+type VideoPopoverMenuProps = {
+    /** Whether  popover menu is visible. */
+    isPopoverVisible: boolean;
```

Comment:
```suggestion
    isPopoverVisible?: boolean;
```

Reply:
Why do you think this change should be applied? 
Even if it was optional previously, I think it should be required  - following [conversion table](https://github.com/Expensify/App/blob/main/contributingGuides/PROPTYPES_CONVERSION_TABLE.md#dont-rely-on-isrequired).

> Regardless of isRequired is present or not on props in PropTypes, read through the component implementation to check if props without isRequired can actually be optional. The use of isRequired is not consistent in the current codebase. Just because isRequired is not present, it does not necessarily mean that the prop is optional.

Please let me know if there is any other reason.